src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -168,4 +175,5 @@
      [system-message-content message-data]
      [user-message-content message-data context keyboard-shown false])
    [reactions/message-reactions-row message-data
-    [user-message-content message-data context keyboard-shown true]]])
+    [rn/view {:pointer-events :none}
```

Comment:
Does clicking a mention and long pressing a message work with `:pointer-events none`?

Reply:
this is for selected-item , it should be inactive

Reply:
Totally, Great work!